### PR TITLE
Divers fix to make pg-backup works on all deployments. ( :warning: ) 

### PR DESCRIPTION
**Pull request by Surfict**
- Readme
- Remove prune while deploying to allow two PG-backup in the same stack (Dalco staging and dalco production)
- Parametrization of the service name for osparc-public

### CODE REVIEW

**Comment by mrnicegyu11** [comment]
( :warning: ): we need to check that the docker node label `pgbackup` is set on the machine that has the tank mounted

File: services/pg-backup/docker-compose.public.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: '3.7'
+services:
+  {{ PREFIX_STACK_NAME }}_pg_backup:
+    volumes:
+    - /var/run/docker.sock:/var/run/docker.sock
```
Comment (mrnicegyu11) [inline services/pg-backup/docker-compose.public.yml.j2:5]: Why does this need to have the docker socket mounted for public?

**Review by mrnicegyu11 — COMMENTED** [review]
I have one question, otherwise thanks a lot looks good

File: services/pg-backup/docker-compose.public.yml.j2 (hunk 1 above)
Comment (Surfict) [reply services/pg-backup/docker-compose.public.yml.j2:5]: It is from the precedent pg-backup. I will remove it. Thanks

**Review by mrnicegyu11 — APPROVED** [review]

**Pull request merged**